test/nebulex/adapters/multilevel_inclusive_test.exs

```diff
@@ -155,6 +156,39 @@ defmodule Nebulex.Adapters.MultilevelInclusiveTest do
     end
   end
 
+  describe "delayed multilevel" do
+    setup_with_dynamic_cache(MultilevelWithDelay, :multilevel_inclusive_with_delay,
+      model: :inclusive,
+      levels: [
+        {MultilevelWithDelay.L1,
+         name: :multilevel_inclusive_with_delay_l1,
+         gc_interval: @gc_interval,
+         backend: :shards,
+         partitions: 2},
+        {MultilevelWithDelay.L2,
+         name: :multilevel_inclusive_with_delay_l2,
+         gc_interval: @gc_interval,
+         backend: :shards,
+         partitions: 2}
+      ]
+    )
+
+    test "does not replicate the data if the cache expires during replication" do
+      # reading from L2 will take 500ms
+      Nebulex.TestCache.DelayedReadAdapter.put_read_delay(500)
```

Comment:
@costaraphael let's alias the module to avoid the credo warning

Reply:
Done in [588d14f](https://github.com/cabol/nebulex/pull/232/commits/588d14fe06334a6d230e7cd8eeda72eec25d761a)